src/scripts/visualizations/tooltip.js

```diff
@@ -97,10 +98,21 @@
                     };
 
                     return clampPosition(pos);
+                },
+
+                'cursor': function cursorPositioner() {
+                    var pos = d3.mouse(layer[0][0]);
+                    var tooltip = tooltipElement[0][0]
+                    var offsetX = 40;
```

Comment:
It's the magic offset number that position the tooltip nicely on the in the horizontal position regardless of the size of the tooltip.